WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsFragment.java

```diff
@@ -1284,6 +1290,31 @@ public void onDismiss(DialogInterface dialog) {
         return view;
     }
 
+    private boolean isValidIpOrRange(String entry) {
+        if (TextUtils.isEmpty(entry)) {
+            return false;
+        }
+
+        // remove whitespace
+        entry = entry.trim();
+        entry = entry.replaceAll(" ", "");
```

Comment:
In that case, we should do something like this:
`entry = entry.replaceAll("\\s","");`

Details can be found in [this answer](https://stackoverflow.com/a/15633290).